CONTRIBUTING.adoc

```diff
@@ -0,0 +1,90 @@
+
+== Enhancements
+Like bug reports, enhancements are submitted through the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page.
+
+As with Bug Reports, please do a search of the open requests first before submitting a new one to avoid duplicates. If you find an existing one, give it a +1.
+
+[NOTE]
+If this is an enhancement you intend to implement, please be sure to submit the enhancement *before* working on any code changes. This will allow members on the workshop team to have a discussion with you to ensure that it's the right design and that it makes sense to include in the workshop.
+
+Enhancements are labeled with link:https://github.com/aws-samples/aws-workshop-for-kubernetes/labels/Enhancement[Enhancement].
+
+=== Submitting an Enhancement
+Open an link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[issue] with the following:
+
+* A short, descriptive title. Ideally, other community members should be able to get a good idea of the enhancement just from reading the title.
+* A detailed description of the the proposed enhancement. Include justification for why it should be added to the workshop, and possibly example code to illustrate how it should work.
+* link:https://guides.github.com/features/mastering-markdown/[Markdown] formatting as appropriate to make the request easier to read.
+* If you intend to implement this enhancement, indicate that you'd like to the issue to be assigned to you.
+
+== Contributions
+Contributions to the workshop are done through link:https://github.com/aws-samples/aws-workshop-for-kubernetes/pulls[Pull Requests]. Please keep the following in mind when considering a contribution:
+
+* The workshop is released under the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/blob/master/LICENSE[Apache 2.0 License]. Any code you submit will be released under this license. If you are contributing a large/substantial enhancement, you may be asked to sign a link:https://github.com/aws/aws-cla[Contributor License Agreement (CLA)].
+
+* For anything but very small or quick changes, you should always start by checking the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page to see if the work is already being done by another person.
+
+[NOTE]
+If you're working on a bug fix, check to see if the bug has already been reported. If it has but no one is assigned to it, ask one of the maintainers to assign it to you before beginning work.  If you're confident the bug hasn't been reported yet, create a new link:#bug-reports[Bug Report] then ask to be assigned to it.
```

Comment:
We may not be able to assign the bug to a user that does not belong to the org

Reply:
Changed to reflect assignment by comment, though this will be harder to track.